ee/ucp/swarm/deploy-to-collection.md

```diff
@@ -57,20 +57,30 @@ services:
 
   wordpress:
     image: wordpress
+    networks:
+      - wp
     ports:
       - 8080:80
     environment:
       WORDPRESS_DB_PASSWORD: example
-    deploy:  
+    deploy:
       labels:
         com.docker.ucp.access.label: /Shared/wordpress
   mysql:
     image: mysql:5.7
+    networks:
+      - wp
     environment:
       MYSQL_ROOT_PASSWORD: example
-    deploy:  
+    deploy:
       labels:
         com.docker.ucp.access.label: /Shared/wordpress
+
+networks:
```

Comment:
I wonder though if the original example intended the network to be created from this stack, or if this was meant to be an external network (created up-front); if so, this page probably needs a description on how to create the network first, and the network in this stack should have `external: true`

Reply:
Agreed, either way, it's ambiguous and not helpful to the reader. I'll let someone from docs comment on which direction they would prefer this to go.